package.json

```diff
@@ -37,6 +37,7 @@
     "superagent-promise": "^0.2.0",
     "taskcluster-base": "^6.0.0",
     "taskcluster-client": "^0.23.8",
+    "taskcluster-lib-api": "^3.0.0",
```

Comment:
This won't be enough, as `src/api.js` uses `base.API`, thus getting whatever version of taskcluster-lib-api is required by taskcluster-base.

That will need to be modified to `require('taskcluster-lib-api')` instead.